tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -53,25 +59,32 @@
 
   @BeforeClass
   public static final void beforeClass() throws Exception {
-    FileSystem fileSystem = sLocalAlluxioClusterResource.get().getClient();
-    FileSystemTestUtils
-        .createByteFile(fileSystem, "/testFile1", WriteType.CACHE_THROUGH, FILE_LEN);
-    FileSystemTestUtils.createByteFile(fileSystem, "/testFile2", WriteType.THROUGH, FILE_LEN);
+    sFileSystem = sLocalAlluxioClusterResource.get().getClient();
   }
 
   @After
-  public final void after() throws Exception {
+  public final void after() throws IOException, AlluxioException {
```

Comment:
The problem is that `LocalAlluxioClusterResource` works as a ClassRule. The configuration can only be reset until all tests are executed. Do I need to change `LocalAlluxioClusterResource` to a TestRule from the ClassRule?